misc/bench/src/yz_driver.erl

```diff
@@ -159,6 +160,17 @@ run(load_fruit, KeyValGen, _, S=#state{iurls=URLs}) ->
         {error, Reason} -> {error, Reason, S2}
     end;
 
+run(load_fruit_plus_spaces, KeyValGen, _, S=#state{iurls=URLs}) ->
+    Base = get_base(URLs),
+    {Key, Val} = KeyValGen(),
+    Key2 = mochiweb_util:quote_plus(lists:concat([?SPACER, Key])),
```

Comment:
```erlang
 list_to_binary(lists:concat([' foo bar', binary_to_integer(<<"44">>)])).
``` 

would then return `<<" foo bar44">>`... which would get formatted correctly on the url calls.